app/controllers/vendor_api/offers_controller.rb

```diff
@@ -0,0 +1,22 @@
+module VendorApi
+  class OffersController < VendorApiController
+    rescue_from ActiveRecord::RecordNotFound, with: :application_not_found
+
+    def create
+      application_choice = ApplicationChoice.find(params[:application_id])
+
+      make_an_offer = MakeAnOffer.new(application_choice: application_choice, offer_conditions: params[:data]).call
+      if make_an_offer.successful?
+        render json: { data: SingleApplicationPresenter.new(make_an_offer.application_choice).as_json }
+      else
+        raise NotImplementedError
+      end
+    end
+
+  private
+
+    def application_not_found(_e)
```

Comment:
This method is duplicated from the `ApplicationsController`, perhaps we can push it up to `VendorApiController`?

Reply:
Also in #267 — we discussed this and planned to let the duplication ride for these two PRs, then refactor